src/utilities/normalizeSnapPoint.ts

```diff
@@ -16,5 +16,5 @@ export const normalizeSnapPoint = (
     normalizedSnapPoint =
       (Number(normalizedSnapPoint.split('%')[0]) * containerHeight) / 100;
   }
-  return Math.max(0, containerHeight - normalizedSnapPoint);
+  return Math.round(Math.max(0, containerHeight - normalizedSnapPoint));
```

Comment:
this will cause an issue on Android , https://github.com/gorhom/react-native-bottom-sheet/issues/168

Reply:
thanks for the context @gorhom, I didn't check the history :D. I thought this could be a tricky thing. I actually went to `v3.0.2-alpha.0` where the bug was reported - and when using `Math.round` instead of `Math.ceil`, the simple modal works fine... but the "dynamic snap points" demo does not... clearly there is some more magic going on. I have reverted this part.